roles/kubernetes/master/tasks/kubeadm-upgrade.yml

```diff
@@ -9,6 +9,7 @@
     --allow-experimental-upgrades
     --allow-release-candidate-upgrades
     --etcd-upgrade=false
+    {{ (kubeadm_output.stdout is version('v1.16.0', '>=')) | ternary('--certificate-renewal=true', '') }}
```

Comment:
I would normally request this be made into a configurable variable, but we need some refactoring on this task. I will refactor this in the next couple of weeks.

Reply:
--certificate-renewal=true is should default behavior... this is bug workaround for 1.16.